api/db.py

```diff
@@ -41,31 +40,39 @@ def get_db_cursor():
         conn.close()
         
 
-def get_data(sql):
+# get table names from database 
+def get_data_tables(limit,offset):
```

Comment:
_"Please keep PEP8":_
Can you tell me more specifically where you found the conventions not to be accordingly to PEP8? you meant in the whole code or just in `get_data_tables(limit,offset):` ?
_"shouldn't the limit and offset have defaults?"_
I set the defaults in `db_tables()` route in `main.py` to limit=0 and offset=0,basically to not return any rows.

Reply:
About PEP8, for example in line 44 the arguments for the function don't have space between the comma and offset. If I remember correctly there were more places that didn't conform to PEP8.
If you prefer we can use flake8 for linting checks and that way we will get errors whenever we don't conform to it.

About the defaults, sounds good, but shouldn't we prefer have some sensible default for the limit (like 10 or 20)? after all the point of the defaults is that we won't have to enter them all the time (for offset 0 is sensible since you want the first page of results to appear if no data was sent regarding that)

Reply:
For first question, yes that's a good idea using `flake8`.
For second question, in all routes except `db` it give the whole table with default limit 0. If you think it's too much and 10 or 20 is enough that's ok we can change it.

Reply:
Do we want to get the whole table? if yes than it's OK.
What is the expectation for the future on number of rows?

Reply:
I don't know what is the expectation for the future.
I think only problem with get the whole table is potential overloading on the database when numerous users want to get data from a very large table. What do you think?